.circleci/config.yml

```diff
@@ -46,6 +68,16 @@ install_dep: &install_dep
         pip install --progress-bar off opencv-python pytest-xdist
         pip install --progress-bar off torch torchvision
 
+install_dep_win: &install_dep_win
+  - run:
+      name: Install Dependencies on Windows
+      command: |
+        pip install cython
```

Comment:
From the current failures in https://circleci.com/gh/facebookresearch/detectron2/1690?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link it looks like `numpy` has to be installed here as well. `fetch_build_eggs` for some reason does not work on windows